src/Bridges/Objective/bridges/slack.jl

```diff
@@ -174,3 +174,76 @@ function MOI.get(
     g = MOI.Utilities.remove_variable(f, bridge.slack)
     return MOI.Utilities.convert_approx(G, g)
 end
+
+"""
+    struct SlackBridgePrimalDualStart <: MOI.AbstractModelAttribute end
+
+[`Bridges.Objective.SlackBridge`](@ref) introduces a new constraint into the
+problem. However, because it is not a constraint bridge, it cannot intercept
+calls to set [`ConstraintDualStart`](@ref) or [`ConstraintPrimalStart`](@ref).
+
+As a work-around, set this attribute to `true` to set the primal and dual
+start for the new constraint. This attribute must be set after
+[`VariablePrimalStart`](@ref).
+"""
+struct SlackBridgePrimalDualStart <: MOI.AbstractModelAttribute end
+
+function MOI.throw_set_error_fallback(
+    ::MOI.ModelLike,
+    ::SlackBridgePrimalDualStart,
+    ::Bool,
+)
+    return # Silently ignore if the model does not support.
+end
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::SlackBridgePrimalDualStart,
+    ::Type{<:SlackBridge},
+)
+    return true
+end
+
+function MOI.set(
+    model::MOI.ModelLike,
+    ::SlackBridgePrimalDualStart,
+    b::SlackBridge{T},
+    value::Bool,
+) where {T}
+    @assert value
+    # ConstraintDual: if the objective function had a dual, it would be `-1` for
+    # the Lagrangian function to be the same.
+    if MOI.supports(model, MOI.ConstraintDualStart(), typeof(b.constraint))
```

Comment:
This does not seem right, we should get an `UnsupportedAttribute` error if it's not supported.
We should implement `supports` for the bridge so that it returns `false` if the constraint type does not support it. That way, it would be ignored by `copy_to` ?

Reply:
We only ignore the `Name` attribute. This is a special case because we want to pretend that every solver supports it.

Reply:
Indeed, maybe drop a comment here so that people see it's not the common way of handling starting values and it's not copy-pasted out of context

Reply:
Done